plugins/versionpress/src/Utils/StringUtils.php

```diff
@@ -26,14 +26,14 @@ public static function verbToPastTense($verb)
     }
 
     /**
-     * Converts LF string to CRLF string
+     * Ensures LF line endings in a string
      *
-     * @param string $str LF line endings
-     * @return string CRLF line endings
+     * @param string $str LF or CRLF line endings
+     * @return string LF line endings
      */
-    public static function crlfize($str)
+    public static function ensureLf($str)
     {
-        return str_replace("\n", "\r\n", str_replace("\r\n", "\n", $str));
+        return str_replace("\\r\\n", "\\n", $str);
```

Comment:
Fixed in https://github.com/versionpress/versionpress/pull/1208/commits/9ac16030da390ad39296c83512f24f92d13447e8.